src/main/java/com/org/jenkins/Custom/Jenkins/Distribution/Service/UpdateCenterService.java

```diff
@@ -0,0 +1,27 @@
+package com.org.jenkins.Custom.Jenkins.Distribution.Service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.apache.commons.io.FileUtils;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UpdateCenterService {
+
+    private static final String UPDATE_CENTER_JSON_URL = "https://updates.jenkins.io/current/update-center.actual.json";
+    
+    public void downloadUpdateCenterJSON() throws IOException {
```

Comment:
Eventually you might want to pull this out to a config file to make it customisable for someone who wants to run the tool against a custom update center.  I'm fine with this being here now.

Reply:
Yeah I agree, so that the user can put in the customized url in a config file and then the file downloader can just use that url